app/Company.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace App;
+
+use Illuminate\Database\Eloquent\SoftDeletes;
+use Illuminate\Foundation\Auth\User as Authenticatable;
+
+class Company extends Authenticatable
+{
+    use SoftDeletes;
+    /**
+     * The attributes that are mass assignable.
+     *
+     * @var array
+     */
+    protected $fillable = [
+        'name',
+        'address',
+        'website',
+    ];
+
+    /**
+     * Get the customers work in company.
+     */
+    public function customers()
+    {
+        return $this->hasMany('App\Customer');
```

Comment:
@hapo-thomnt có thể khai báo đơn giản hơn `return $this->hasMany(Customer::class);
`